.travis.yml

```diff
@@ -3,19 +3,17 @@ lein: 2.7.1
 sudo: false
 
 install:
-  - (cd cloverage && lein deps)
-  - (cd lein-cloverage && lein deps)
   # Get recent node:
   - . $HOME/.nvm/nvm.sh
   - nvm install stable
   - nvm use stable
-  - npm install
+  - npm install -g eclint
 
 before_script:
-  - npm install -g eclint
-  - eclint check .* **
-  - (cd cloverage && lein cljfmt check)
-  - (cd lein-cloverage && lein cljfmt check)
+  - eclint check .* *
+  - (cd cloverage && lein do deps, cljfmt check)
+  - (cd lein-cloverage && lein do deps, cljfmt check)
```

Comment:
What is the purpose of `lein do deps, ...`? Why doesn't `lein cljfmt check` just do the right thing?

Reply:
Yes, I can drop the `do deps, ` part.  It's redundant.

Reply:
Please do :)